src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/RequestPostHearingTypes.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import static uk.gov.hmcts.reform.sscs.ccd.domain.DwpState.SET_ASIDE_REQUESTED;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.REQUEST_CORRECTION;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.REQUEST_LIBERTY_APPLY;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.REQUEST_PERMISSION_APPEAL;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.REQUEST_SET_ASIDE;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.REQUEST_SOR;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum RequestPostHearingTypes implements CcdCallbackMap {
+    SET_ASIDE("setAside","Set Aside", REQUEST_SET_ASIDE, "Set Aside Request Made", "Set Aside Request Made", SET_ASIDE_REQUESTED),
+    CORRECTION("correction","Correction", REQUEST_CORRECTION, "Correction Request Made", "Correction Request Made", null),
+    STATEMENT_OF_REASONS("statementOfReasons","Statement of Reasons", REQUEST_SOR, "Statement of Reasons Request Made", "Statement of Reasons Request Made", null),
```

Comment:
lines too long

Reply:
We have been ignoring line length on the enums as it makes them hard to read otherwise